cpu/rp2350/Makefile

```
@@ -0,0 +1,13 @@
+MODULE = cpu
+
+USEPKG += picosdk
+
+DIRS = $(RIOTCPU)/cortexm_common periph
```

```suggestion
DIRS += $(RIOTCPU)/cortexm_common
DIRS += periph
```

This makes sure that the `DIRS` variable is not overriden in case something else sets it.
The two lines are just cosmetic.